src/prepare.rs

```diff
@@ -97,7 +101,7 @@ pub fn isolating_run_sequences(
             };
 
             isolating_run_sequences.push(IsolatingRunSequence {
-                runs: vec![run],
+                runs: vec![run.clone()],
```

Comment:
we can use `.drain()` or `into_iter` so we can move instead

Reply:
Actually, we can just remove the `.clone()`, it's not needed here.